lib/dogwatch/model/response.rb

```diff
@@ -11,32 +11,42 @@ class Response
       ERROR = '400'.freeze
       CREATED = '200'.freeze
       ACCEPTED = '202'.freeze
+
       colorize(:action,
                :green => [:created, :accepted, :updated],
                :yellow => [],
                :red => [:error])
 
       attr_accessor :response
 
-      def initialize(response, updated = false)
+      # @param [Array] response
+      # @param [String] name
+      # @param [TrueClass|FalseClass] updated
```

Comment:
This is typically `Boolean` in yardoc.

http://www.rubydoc.info/gems/yard/file/docs/Tags.md#Class_or_Module_Types